--- pixman-0.42.2/test/Makefile.in
+++ new/test/Makefile.in
@@ -289,7 +289,7 @@
 	$(top_builddir)/pixman/libpixman-1.la $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1)
 region_test_SOURCES = region-test.c
-region_test_OBJECTS = region-test.$(OBJEXT)
+region_test_OBJECTS = region-test.$(OBJEXT) utils.$(OBJEXT)
 region_test_LDADD = $(LDADD)
 region_test_DEPENDENCIES = libutils.la \
 	$(top_builddir)/pixman/libpixman-1.la $(am__DEPENDENCIES_1) \
@@ -319,7 +319,7 @@
 	$(top_builddir)/pixman/libpixman-1.la $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1)
 scaling_helpers_test_SOURCES = scaling-helpers-test.c
-scaling_helpers_test_OBJECTS = scaling-helpers-test.$(OBJEXT)
+scaling_helpers_test_OBJECTS = scaling-helpers-test.$(OBJEXT) utils.$(OBJEXT)
 scaling_helpers_test_LDADD = $(LDADD)
 scaling_helpers_test_DEPENDENCIES = libutils.la \
 	$(top_builddir)/pixman/libpixman-1.la $(am__DEPENDENCIES_1) \
